tests/test_connections_page.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+from utilities.data_randomizer import generate_invalid_invitation_code
+
+
+@pytest.mark.smoke
+@pytest.mark.regression
+def test_update_connection_status(go_to_connections_page):
+    connections_page = go_to_connections_page
+    # activate > deactivated
+    active_status = connections_page.get_connection_status()
+    deactivated_status = connections_page.invoke_dropdown_menu().deactivate_connection().get_connection_status()
+    assert deactivated_status != active_status, 'Status is not updated'
+    # deactivated > active
+    reactivated_status = connections_page.invoke_dropdown_menu().activate_connection().get_connection_status()
+    assert reactivated_status != deactivated_status, 'Status is not updated'
+    assert reactivated_status == active_status, 'Status is not updated'
+
+
+def test_default_invitation_dialog(go_to_connections_page):
+    connections_page = go_to_connections_page
+    connections_page.is_accept_code_dialog_shown()
```

Comment:
what is the purpose of this line if you do not write the result to a variable and never use it?

Reply:
Agree. Updated method of PageObject > get modal dialog. Since it will be useful for further checks of dialog. Currently existing TCs were updated respectively.